pyproject.toml

```diff
@@ -58,7 +58,7 @@ mypy = {version = "^0.971", optional = true}
 black = {version = "^22.8.0", optional = true}
 isort = {version = "^5.10.1", optional = true}
 flake8 = {version = "^5.0.4", optional = true}
-pylint = {version = "^2.12.2", optional = true}
+pylint = {version = "^2.15.4", optional = true}
 coverage = {version = "*", optional = true}
 screenpy-adapter_allure = {version = "^4.0.1", optional = true}
 screenpy-appium = {version = "*", optional = true}
```

Comment:
I bumped the python version to avoid version dependency conflicts.  Poetry was showing the the following when I attempted to use it's dependency handling.

```
The current project's Python requirement (>=3.7,<4.0) is not compatible with some of the required packages Python requirement:
  - pylint requires Python >=3.7.2, so it will not be satisfied for Python >=3.7,<3.7.2

Because no versions of pylint match >2.15.4,<3.0.0
 and pylint (2.15.4) requires Python >=3.7.2, pylint is forbidden.
So, because screenpy depends on pylint (^2.15.4), version solving failed.
```